config/webpack.config.preact.js

```diff
@@ -12,6 +14,8 @@ module.exports = {
   },
   resolve: {
     extensions: ['.jsx'],
+    // necessary so that webpack looks into node_modules for preact-compat when importing cozy-client
```

Comment:
I am surprised of this. Webpack and NPM should do that natively if preact-compat is declared in cozy-client. What is the specificity ?

Reply:
Justement, idéalement je ne voudrais pas mettre preact-compat en dépendance de `cozy-client`, mais react, afin de rester compatible. Et je n'ai pas trouvé de meilleure solution que ça.

Reply:
I'm still surprised that the whole library depends on (p)react. If I want to use `cozy-client` for nodejs konnector, I don't want to `npm install react` or `npm install preact preact-compat`. Is that a concern?